### PR TITLE
Update README with Status App Functionality Demo

### PR DESCRIPTION
**Pull request by FFFra**
### Summary
Add Status App Functionality Demo to docs

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@
 
 [re-frame usage (recorded meeting)](https://drive.google.com/file/d/1qv_E0CEGzQpu_zGXD0gCTU5EvhC2k8Jy/view)
 
+[status app functionality demo](https://drive.google.com/file/u/0/d/1PmwsMLTLDNNIdv5_6wvMOwoj2PfT50c6/view?usp=drive_web&pli=1)
```
Comment (vkjr) [inline doc/README.md]: Just a nitpick) I'd remove the url parameters from the link `?usp=drive_web&pli=1` to make it clearer)

**Review by vkjr — APPROVED** [review]

File: doc/README.md (hunk 1 above)
Comment (FFFra) [reply doc/README.md]: Thank you, I did not noticed that

File: doc/README.md [hunk 2]
```diff
@@ -41,6 +41,8 @@
 
 [re-frame usage (recorded meeting)](https://drive.google.com/file/d/1qv_E0CEGzQpu_zGXD0gCTU5EvhC2k8Jy/view)
 
+[status app functionality demo](https://drive.google.com/file/u/0/d/1PmwsMLTLDNNIdv5_6wvMOwoj2PfT50c6/view)
```
Comment (J-Son89) [inline doc/README.md:44]: does it nee d a quick explanation of what the video is 🤔

**Review by J-Son89 — APPROVED** [review]

**Comment by J-Son89** [comment]
good stuff @FFFra! :)

**Pull request merged**